registry/scripts/deploy.ts

```diff
@@ -0,0 +1,14 @@
+import { ethers } from "hardhat";
+
+async function main() {
+  console.log("Importing the deploy script");
+  await import("@tableland/evm/scripts/deploy.js");
```

Comment:
The `@tableland/evm` deploy script will run as soon as it's imported so we need to do a dynamic import inside the main function to ensure it's not run before the hardhat node is ready.